Source/WebInspectorUI/UserInterface/Models/EventBreakpoint.js

```diff
@@ -132,8 +183,11 @@ WI.EventBreakpoint = class EventBreakpoint extends WI.Breakpoint
     saveIdentityToCookie(cookie)
     {
         cookie["event-breakpoint-type"] = this._type;
-        if (this._eventName)
+        if (this._eventName) {
             cookie["event-breakpoint-event-name"] = this._eventName;
+            cookie["event-breakpoint-case-sensitive"] = this._caseSensitive;
+            cookie["event-breakpoint-is-regex"] = this._isRegex;
```

Comment:
Nothing ever seems to use any of these cookie values?

Reply:
these are used by the `WI.ContentView` routing system to know whether it can re-use an old `WI.ContentView`, as well as comparing `representedObject` to see if the same view can be used (i.e. reselecting the tree element)